evaluator-llm/src/evaluator_llm/app.py

```diff
@@ -0,0 +1,99 @@
+import logging
```

Comment:
FYI for the app i'm not going to review in detail at all, I trust you as you've teamed on it, I'll focus on the integration / helm / docker etc side of things! So i might miss stuff here